src/gui/src/displayControls.cpp

```diff
@@ -171,17 +171,20 @@ DisplayControls::DisplayControls(QWidget* parent)
 
   // Nets group
   auto nets_parent = makeItem(
-      nets_group_, "Nets", model_, Qt::Checked, [this](bool visible) {
+      nets_group_,
+      "Nets", model_,
+      Qt::Checked,
+      [this](bool visible) {
         toggleAllChildren(visible, nets_group_.name, Visible);
+      },
+      [this](bool selectable) {
+        toggleAllChildren(selectable, nets_group_.name, Selectable);
       });
 
-  makeItem(nets_.signal, "Signal", nets_parent, Qt::Checked);
-
-  makeItem(nets_.power, "Power", nets_parent, Qt::Checked);
-
-  makeItem(nets_.ground, "Ground", nets_parent, Qt::Checked);
-
-  makeItem(nets_.clock, "Clock", nets_parent, Qt::Checked);
+  makeItem(nets_.signal, "Signal", nets_parent, Qt::Checked, [](bool) {}, [](bool) {});
+  makeItem(nets_.power, "Power", nets_parent, Qt::Checked, [](bool) {}, [](bool) {});
+  makeItem(nets_.ground, "Ground", nets_parent, Qt::Checked, [](bool) {}, [](bool) {});
+  makeItem(nets_.clock, "Clock", nets_parent, Qt::Checked, [](bool) {}, [](bool) {});
```

Comment:
Why specify the null functions as they are optional args.

Reply:
I supposed I didn't need to make the first `[](bool) {}` and could have been `std::function<bool>()`, the second one is needed to get the selection checkbox added. I'll change the first argument to be consistent with how it was done when adding the layers.

Reply:
I think the real problem is that makeItem is doing double duty making leaf items and also group items.  I think if we split it into two methods each would be simpler.  I'm ok with just making it work in this PR and doing that later or folding that change in here.  What do you prefer?

Reply:
I think it would make sense to split it in two. Since the other PR for instances contain the same sin, maybe it's easier to finish and merge in one of them and them make the change in the other one (that way it only needs to be fixed once).